cost/azure/schedule_instance/azure_schedule_instance.pt

```diff
@@ -403,10 +547,14 @@ script "js_instances_with_schedule", type: "javascript" do
     }
 
     // Check that we have a schedule and it is not explicitely disabled before attempting to parse the schedule tag value
-    if (schedule != null) {
-      // schedule="8:00-17:30;MO,TU,WE,TH,FR;America/Los_Angeles"
+    // Test the regex pattern for a valid schedule as well, in case the tag value is not a valid schedule
+    // schedule="08:00-17:30;MO,TU,WE,TH,FR"
+    // schedule="08:00-17:30;MO,TU,WE,TH,FR;America/Los_Angeles"
+    var schedule_is_valid = /([0-9]{1,2}.*)-([0-9]{1,2}.*);([A-Za-z,]*).*/.test(schedule)
+    if ((_.isString(schedule)) && (schedule_is_valid == false)) {
+      console.log("Skipping instance " + instance['resourceID'] + " with invalid schedule '" + schedule +"'");
```

Comment:
Same as `console.log()` comments on AWS policy above

Reply:
Going to use this thread to discuss.. I am was thinking this would be a valid piece of information to have in the logs.  Another way we could return this is including the instance in the results, but with a value for "schedule_error" field or something -- and only resources that have no schedule_error value would be valid and have it's schedule executed.

Or, we leave this as a console.log and ignore this test warning...   any thoughts @XOmniverse ?  We've discussed console.log exemptions before

Reply:
We should probably have a broader conversation about this. I originally added that test because I frequently (mostly in older and poorly maintained policy templates) found simple debugging console.log() statements that were left over without being properly removed from the production template.

If we believe there are things we would want logged via console.log() that are not for debugging during policy development, that test can be disabled.

Reply:
Spoke with Shawn and we added ability to exclude certain console.log lines if we believe they should be kept [336cce3]